app/routes/admin/messages/list.js

```diff
@@ -7,9 +7,9 @@ export default class extends Route.extend(EmberTableRouteMixin) {
     const searchField = 'recipient';
     const filterOptions = this.applySearchFilters([], params, searchField);
     let queryString = {
-      filter        : filterOptions,
-      'page[size]'  : params.per_page || 100,
-      'page[number' : params.page || 1
+      filter         : filterOptions,
+      'page[size]'   : params.per_page || 100,
+      'page[number]' : params.page || 1
```

Comment:
What does this change?

Reply:
I was looking into https://github.com/fossasia/open-event-frontend/issues/3693 then I found the wrong syntax `page[number`

Reply:
the problem with #3693 is due to wrong passing of querystring. I have some doubts regarding that to discuss in technical meeting.